cloud_dashboard/src/molecules/D3HorizonChartSampleBlock.tsx

```diff
@@ -0,0 +1,801 @@
+        <input type="text" className="form-control" id="chartTitle" value={title} onChange={(e) => {
+          setTitle(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartHeight">Height</label>
+        <input type="number" className="form-control" id="chartHeight" value={height} onChange={(e) => {
+          setHeight(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartWidth">Width</label>
+        <input type="number" className="form-control" id="chartWidth" value={width} onChange={(e) => {
+          setWidth(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartYRangeMin">yRangeMin</label>
+        <input type="number" className="form-control" id="chartYRangeMin" value={yRangeMin} onChange={(e) => {
+          setYRangeMin(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartYRangeMax">yRangeMax</label>
+        <input type="number" className="form-control" id="chartYRangeMax" value={yRangeMax} onChange={(e) => {
+          setYRangeMax(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartTitleColor">TitleColor</label>
+        <input type="text" className="form-control" id="chartTitleColor" value={titleColor} onChange={(e) => {
+          setTitleColor(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartTitleFontSize">TitleFontSize</label>
+        <input type="number" className="form-control" id="chartTitleFontSize" value={titleFontSize} onChange={(e) => {
+          setTitleFontSize(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <label htmlFor="chartColors">Colors</label>
+        <select ref={colorsRef} multiple className="form-control" style={{ marginLeft: 0, marginRight: 0 }}>
+          {
+            colors.map((c, index) => {
+              return <option key={index}>{c}</option>
+            })
+          }
+        </select>
+      </div>
+      <div className="form-group">
+        <input type="text" className="form-control" id="chartColors" value={newColor} onChange={(e) => {
+          setNewColor(e.currentTarget.value);
+        }} />
+      </div>
+      <div className="form-group">
+        <button type="button" className="btn btn-default" style={{ marginRight: 20 }} onClick={() => {
+          setColors([...colors, newColor]);
+        }}>追加</button>
+        <button type="button" className="btn btn-default" onClick={() => {
+          const temp = colorsRef.current;
+          if (temp === null) {
+            return;
+          }
+          const newColors: string[] = [];
+          for (let i = 0; i < temp.options.length; i += 1) {
```

Comment:
@RyoYamashita765 This can be:

```suggestion
          for (let i = 0; i < temp.options.length; i++) {
```